dist/en/integrations/shopify/shopify_2024-07_to_shopify_2024-10.json

```diff
@@ -0,0 +1,217 @@
+{
+  "version": "2024-10-unstable",
+  "mappings": [
+    {
+      "input_taxonomy": "shopify/2024-10-unstable",
+      "output_taxonomy": "google/2021-09-21",
+      "rules": [
+
+      ]
+    },
+    {
+      "input_taxonomy": "shopify/2022-02",
+      "output_taxonomy": "shopify/2024-10-unstable",
+      "rules": [
+
+      ]
+    },
```

Comment:
Something things wrong with the generate mappings command. I wouldn't expect other mappings to show in the file specific to 2024-07 -> 2024-10

@chesterbot01